### DISCUSSION TITLE
Overview of ParameterTypes

### DISCUSSION THREAD

**Discussion by SHogenboom** [jspsych/jsPsych#3044] [Q&A] [answered]
Is there an overview of the available `jspsych.ParameterType`s? Ideally accompanied by a description of best practices in terms of when to use which one?  Have been searching but was unable to find it.

It would be pretty useful as I’m trying to develop a plugin from scratch (although according to the provided template).

Thanks in advance!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @SHogenboom

There's not good documentation of this yet. I'm going to open an issue to make sure we add this.

You can find the list of parameter types in this code chunk:

https://github.com/jspsych/jsPsych/blob/a7e6f8c40f4efe2184e71bb1421f07e54a63362a/packages/jspsych/src/modules/plugins.ts#L18-L34
